### DISCUSSION TITLE
iam joining github education using hack club and got rejection

### DISCUSSION THREAD

**Discussion by koko9xxx** [community/community#147856] [GitHub Education]
### Hi everyone,

iam joining github education using hack club and got rejection  why ?

**Comment by VIPSHAKH** [comment]
Your GitHub Education application via Hack Club might have been rejected due to eligibility issues, incomplete information, Hack Club verification problems, or previous misuse. Check the rejection email for details and contact support if needed.

**Comment by koko9xxx** [comment]
Hey kirolos gerges ayad,Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Dec 24, 2024 for @koko9xxx.The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect email domain, please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.Please ensure that your academic affiliation document contains your last name exactly as it appears in your GitHub billing information. If you have only a single legal name, enter it in both the first and last name fields of your GitHub billing information.The image you selected does not appear to contain your school name. Your complete school name must appear in your document, not only the school logo. You may include multiple documents together. If your official document is not in English then you may photograph the original next to an English translation.You appear not to be near any campus location for the school you have selected. Your academic affiliation documentation must indicate virtual learning if you are remote. You must configure your browser and operating system to allow location access. You may not use a VPN. If you believe that our information about your school is incomplete or incorrect e.g. missing campus location(s), then please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.Once you have resolved the issues with your sumbission, visit us again and reapply.Have questions? Please seek help via our Student Developer Pack Application FAQ or visit the GitHub Education Community. The GitHub Education Team
--



</body>
</html>Hey kirolos gerges ayad,

Unfortunately, we were unable to verify your academic affiliation based on the information you submitted on Dec 24, 2024 for [@koko9xxx](https://github.com/koko9xxx).

The school you selected does not appear to have a campus location in your country. Your school-provided academic affiliation document must indicate virtual learning if you are remote. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect email domain, please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.

Please ensure that your academic affiliation document contains your last name exactly as it appears in your [GitHub billing information](https://github.com/settings/billing/payment_information). If you have only a single legal name, enter it in both the first and last name fields of your GitHub billing information.

The image you selected does not appear to contain your school name. Your complete school name must appear in your document, not only the school logo. You may include multiple documents together. If your official document is not in English then you may photograph the original next to an English translation.

You appear not to be near any campus location for the school you have selected. Your academic affiliation documentation must indicate virtual learning if you are remote. You must configure your browser and operating system to allow location access. You may not use a VPN. If you believe that our information about your school is incomplete or incorrect e.g. missing campus location(s), then please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.


Once you have resolved the issues with your sumbission, [visit us again and reapply](https://education.github.com/).

Have questions? Please seek help via our [Student Developer Pack Application FAQ ](https://github.com/orgs/community/discussions/111352)or visit the [GitHub Education Community.](https://github.com/orgs/community/discussions/categories/github-education)

💖 The GitHub Education Team

**Comment by koko9xxx** [comment]
this is email i got

**Comment by koko9xxx** [comment]
the problem that this issues not exist

**Comment by VIPSHAKH** [comment]
GitHub couldn't verify your academic affiliation. To fix this:

Ensure the document includes your full school name and matches your GitHub billing name.
If you're remote, state "virtual learning" in the document.
Enable location access and disable any VPN.

**Comment by elijahrs-gh** [comment]
I have the same problem, I sent a support ticket to Github to fix the Hack Club process, as it seems to be erroring out students because it is looking for your student ID to have 'Hack Club' on it, which they don't because its not one school.

**Reply by koko9xxx** [reply]
So what will they do ?
